ArduCopter/sensors.cpp

```diff
@@ -387,6 +387,10 @@ void Copter::update_sensor_status_flags(void)
         control_sensors_enabled |= MAV_SYS_STATUS_SENSOR_BATTERY;
     }
 
+    if (copter.g2.proximity.sensor_enabled()) {
```

Comment:
This needs to be surrounded by the PROXIMITY_ENABLED macro.

Reply:
Fixed.